pkg/auth/userregistry/identitymapper/provision.go

```diff
@@ -11,41 +8,15 @@ import (
 	"k8s.io/kubernetes/pkg/util/sets"
 
 	authapi "github.com/openshift/origin/pkg/auth/api"
-	"github.com/openshift/origin/pkg/user"
 	userapi "github.com/openshift/origin/pkg/user/api"
 	identityregistry "github.com/openshift/origin/pkg/user/registry/identity"
 	userregistry "github.com/openshift/origin/pkg/user/registry/user"
 )
 
-// UserNameGenerator returns a username
-type UserNameGenerator func(base string, sequence int) string
-
-var (
-	// MaxGenerateAttempts limits how many times we try to find an available username for a new identity
-	MaxGenerateAttempts = 100
-
-	// DefaultGenerator attempts to use the base name first, then "base2", "base3", ...
-	DefaultGenerator = UserNameGenerator(func(base string, sequence int) string {
-		if sequence == 0 {
-			return base
-		}
-		return fmt.Sprintf("%s%d", base, sequence+1)
-	})
-)
-
 type provisioningIdentityMapper struct {
```

Comment:
doc this.  It provisions identities if needed, but not may or may not provision users and identity to user mappings depending on strategy.  The name is deceptive/difficult.